### PR TITLE
refactor(rxBulkSelectUtils): FRMW-390-convert to Quark Service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-390

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `rxBulkSelect.rxBulkSelectUtils` to Quark Service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `rxBulkSelectUtils` Quark Service

### Demo Page
screen shot 2016-01-19 at 1 28 01 pm

### API Docs
screen shot 2016-01-19 at 1 28 22 pm

**Comment by 100stacks** [comment]
Merge after PR#1501.

**Comment by 100stacks** [comment]
Merge after PR #1501.

**Comment by parlarjb** [comment]
LGTM

File: src/components/rxBulkSelect/rxBulkSelect.module.js [hunk 1]
```diff
@@ -13,11 +13,8 @@
  * * {@link rxBulkSelect.directive:rxBulkSelectHeaderCheck rxBulkSelectHeaderCheck}
  * * {@link rxBulkSelect.directive:rxBulkSelectRow rxBulkSelectRow}
  * * {@link rxBulkSelect.directive:rxBulkSelectValidate rxBulkSelectValidate}
- *
- * ## Services
- * * {@link rxBulkSelect.service:NotifyProperties NotifyProperties}
- * * {@link rxBulkSelect.service:rxBulkSelectUtils rxBulkSelectUtils}
  */
 angular.module('encore.ui.rxBulkSelect', [
+    'encore.ui.quarks',
```
Comment (CITguy) [inline src/components/rxBulkSelect/rxBulkSelect.module.js]: FYI, this file is likely to conflict with #1501. Merge this PR *after* 1501.

**Pull request merged**